velox/functions/sparksql/String.h

```diff
@@ -283,6 +283,82 @@ struct EndsWithFunction {
   }
 };
 
+/// substring_index function
+/// substring_index(string, string, int) -> string
```

Comment:
The signature is obvious from the code. Let's not duplicate it here.

Reply:
Removed.